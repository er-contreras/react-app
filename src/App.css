* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

html {
  font-size: 62.5%;
}

.App {
  min-height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
}

.active-link {
  color: orangered;
  text-decoration: underline;
}
