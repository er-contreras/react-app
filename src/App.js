import React from 'react';
import './App.css';
import { HashRouter as Router, Route, Switch } from 'react-router-dom';
import Calculator from './components/Calculator';
import Home from './pages/Home';
import Quote from './pages/Quote';
import Navbar from './components/Navbar';

const App = () => (
  <>
    <Navbar />
    <Router>
      <Switch>
        <Route exact path="/home">
          <Home />
        </Route>

        <Route path="/calculator">
          <div className="App">
            <Calculator />
          </div>
        </Route>

        <Route path="/quote">
          <Quote />
        </Route>
      </Switch>
    </Router>
  </>
);

export default App;
