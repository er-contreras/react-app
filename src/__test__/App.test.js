import React from 'react';
import TestRenderer from 'react-test-renderer';
import App from '../App';

describe('App test', () => {
  it('Should render same text passed into ...', () => {
    const tree = TestRenderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})