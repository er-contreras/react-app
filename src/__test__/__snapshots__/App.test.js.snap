// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test Should render same text passed into ... 1`] = `
<nav
  className="navBar"
>
  <div
    id="title"
  >
    <h1>
      Math Magicians
    </h1>
  </div>
  <ul>
    <li>
      <a
        aria-current={null}
        href="#/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="#/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="#/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
